doc/api/deprecations.md

```diff
@@ -0,0 +1,574 @@
+# Deprecated APIs
+
+Node.js may deprecate APIs when either: (a) use of the API is considered to be
+unsafe, (b) an improved alternative API has been made available, or (c)
+breaking changes to the API are expected in a future major release.
+
+Node.js utilizes two kinds of Deprecations:
+
+* Documentation-only
+* Runtime
+
+A Documentation-only deprecation is one that is expressed only within the
+Node.js API docs. These generate no side-effects while running a Node.js
+process.
+
+A Runtime deprecation will, by default, generate a process warning that will
```

Comment:
"process warning" as opposed to some other kind of warning? Or do you mean, "emit a `'warning'` event on process"?